hera_pspec/tests/test_uvpspec.py

```diff
@@ -615,6 +620,50 @@ def test_combine_uvpspec(self):
         out = uvpspec.combine_uvpspec([uvp_a, uvp_b], merge_history=False, verbose=False)
         assert 'batwing' in out.history and not 'foobar' in out.history
```

Comment:
Do we need another test for uvps have different cov_array and stats_array keys so that the combined uvp doesn't have  such two arrays?

Reply:
Hmm, not quite sure I know what you mean. cov_array is an ndarray while `stats_array` is a dictionary, so it is expected that they can in principle hold different kinds of information--even if we are using stats_array as the sqrt(diagonal(cov_array)) in practice. Can you give me an example?

Reply:
Yes. I mean for uvp1 we have a cov_model 'model1', while for uvp2 we have a cov_model 'model2', thus their combined uvp should not have the attribute cov_array as required by the code.

Reply:
Ah, I see what you are saying. Yes, that is probably a good test! thanks